packages/ra-ui-materialui/src/list/Datagrid.js

```diff
@@ -110,6 +110,7 @@ function Datagrid({ classes: classesOverride, ...props }) {
         setSort,
         size = 'small',
         total,
+        selectableRows = () => true,
```

Comment:
I think this name is confusing for a function as it does not contain a verb. Moreover, this function is called with a single row, so `Rows` is confusing too here. What about `canSelectRow` ?

Reply:
You are right, i'll change it.

Reply:
Done!